docs/source/conf.py

```diff
@@ -93,3 +93,8 @@
 # Should only be uncommented when testing page development while notebooks
 # are breaking.
 # nbsphinx_allow_errors = True
+
+
+# documentation of classes
+# https://stackoverflow.com/questions/5599254/how-to-use-sphinxs-autodoc-to-document-a-classs-init-self-method
+# autoclass_content = 'both'
```

Comment:
tried something. didnt work out.

Reply:
You can probably delete this if we don't need it.